tests/views/test_path.py

```diff
@@ -14,12 +14,15 @@ def test_invalid_repo_name(self):
         self.assertEqual(parsed_response_data['errors'][0]['message'], 'Not Found')
 
     def test_valid_repo_name_for_manifest(self):
-        headers = {'Accept': 'application/vnd.docker.distribution.manifest.v2+json'}
-        response = self.test_client.get('/v2/redhat/foo/manifests/1.25.1-musl', headers=headers)
+        # #3303: verify multi-valued headers too
+        # manifest lists are evaluated first, so pass a longer media type that
+        # matches the manifest list as a prefix
+        headers = {'Accept': 'application/vnd.docker.distribution.manifest.list.v2+jsonjunk,application/vnd.docker.distribution.manifest.v2+json'}  # noqa
```

Comment:
i don't think +jsonjunk would be a valid mediatype sent from docker client :) max what you can have is +json or +prettyjws

Reply:
That's with a civilized client. As a server, you should always check the inputs from the client. This is not a realistic example, just like this patch is not fixing something broken *now*.